models/issues/content_history.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package issues
+
+import (
+	"context"
+	"fmt"
+
+	"code.gitea.io/gitea/models/db"
+	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/timeutil"
+
+	"xorm.io/builder"
+)
+
+// IssueContentHistory save issue/comment content history revisions.
+type IssueContentHistory struct {
```

Comment:
How about use `ContentHistory` as struct and keep database name as `issue_content_history`?